doc: include link to releases (#52) (#53)
rhenning/resume: README.md

limit the toil and manual process associated with my daily work.

When _Fastnet_—a Philadelphia-area ISP and hosting provider I very much respected—
was seeking out Linux sysadmins, I jumped at the opportunity, had an interview,
and was thrilled to receive an offer letter, which I graciously accepted.

I learned _so much_ during my time at Fastnet, remaining there for about eight years—
the longest stint of my career—and surviving the dot-com boom, bust, a merger
with _NetAxs_—another well-regarded Philly-area ISP—and two follow-up acquisitions.
It was there I learned to apply the fundamentals of what would later become known
as _DevOps_ and _Pair Programming_, sharing in these prototype cultures before those
terms were in vogue. It wasn't out of deference to any sort of ideology; software
automation and version control enabled us to reliably build, provision, operate and
scale the infrastructure and services needed to support a rapidly-growing business,
with slim budgets and a small staff.

That experience left quite an impression, for which I owe a great debt of gratitude
to my mentors and colleagues, several of whom have become lifelong friends.

> Since that time I've had the fortunate to work for a number of forward-looking tech

Author: ```suggestion
Since that time I've been fortunate to work for a number of forward-looking tech
```
